pkg/trimaran/handler_test.go

```diff
@@ -0,0 +1,47 @@
+package trimaran
+
+import (
+	"github.com/stretchr/testify/assert"
+	st "k8s.io/kubernetes/pkg/scheduler/testing"
+	"testing"
+	"time"
+)
+
+func TestHandlerCacheCleanup(t *testing.T) {
+	testNode := "node-1"
```

Comment:
Can we check the test cases to table-driven style

Reply:
I added comments for sub test cases.